fixtures/unions.json

```diff
@@ -1,9 +1,33 @@
 [
 {
     "fields" : {
```

Comment:
Forstår jeg det rigtigt hvis jeg tænker at lave en fixture med alle foreningerne og så en data migration der sørger for at alle afdelinger peger på en forening? 

Eller skal det begge være fixtures?